Taskfile.yml

```diff
@@ -51,6 +52,17 @@ tasks:
     deps:
       - go:update
 
+  iso:
+    desc: Build stboot iso image
+    deps:
+      - image
+    sources:
+      - "{{.IMAGE}}"
+    generates:
+      - "{{.ISO}}"
+    cmds:
+      - "xorriso -as mkisofs -R -f -V STBOOT -e {{.IMAGE}} -no-emul-boot -o {{.ISO}} ."
```

Comment:
Hmm, this will include everything in system-transparency.git, due to the trailing `.`.
```
$ ls -lh out/stboot.iso 
-rw-rw-r-- 1 bjorn bjorn 2,8G jan 11 16:23 out/stboot.iso
```
Yikes! :-)

We need something like
```
mkdir -p $workdir/iso
cp out/artifacts/efi_boot_partition.vfat $workdir/iso/stboot.img
xorriso -as mkisofs -R -f -V STBOOT -e stboot.img -no-emul-boot -o $outputdir/stboot.iso $workdir/iso
```
i.e. construct a directory which is the iso-root.

Reply:
ups :sweat_smile: thanks